src/ImageGallery.jsx

```diff
@@ -539,14 +541,20 @@ export default class ImageGallery extends React.Component {
     return Math.abs(this.state.offsetPercentage) > this.props.swipeThreshold;
   }
 
-  _handleSwiping = (e, deltaX, deltaY, delta) => {
-    const { galleryWidth, isTransitioning, scrollingUpDown } = this.state;
+  _handleSwiping = ({ event, absX, dir }) => {
+    if (this.props.disableSwipe) return;
+    const { galleryWidth, isTransitioning, scrollingUpDown, scrollingLeftRight } = this.state;
     const { swipingTransitionDuration } = this.props;
-    this._setScrollDirection(deltaX, deltaY);
+    this._setScrollDirection(dir);
+    if (this.props.stopPropagation) event.stopPropagation();
+    if (
+      (this.props.preventDefaultTouchmoveEvent || scrollingLeftRight)
+      && event.cancelable
+    ) event.preventDefault();
     if (!isTransitioning && !scrollingUpDown) {
-      const side = deltaX < 0 ? 1 : -1;
+      const side = dir === RIGHT ? 1 : -1;
 
-      let offsetPercentage = (delta / galleryWidth * 100);
+      let offsetPercentage = (absX / galleryWidth * 100);
```

Comment:
`delta` here in [react-swipeable v4](https://github.com/dogfessional/react-swipeable/tree/4.x#event-props) was actually `absX` so updated to use `absX` in v5.